read_test.go

```diff
@@ -218,8 +218,10 @@ func TestReadNativeFrames(t *testing.T) {
 		Name              string
 		existingData      Dataset
 		data              []uint16
+		dataBytes         []byte
 		expectedPixelData *PixelDataInfo
 		expectedError     error
+		pixelLength       uint32
```

Comment:
consider renaming to make it more clear how this operates and when it should be specified, perhaps:

```suggestion
		pixelVLOverride       uint32
```